react-blogs/EzequielCaste-9004/src/App.jsx

```diff
@@ -0,0 +1,25 @@
+import Header from './components/Header';
+import Card from './components/Card';
+import {cardData as data} from './data.js';
+import Footer from './components/Footer';
+import sortByDate from './utils/sortByDate';
+
+function App() {
+  return (
+    <div className="container">
+      <Header />
+
+      <div className="card-container">
+        {
+          data.sort(sortByDate).map((cardData) => 
+            <Card key={cardData.date} data={cardData} />
+          )
```

Comment:
```suggestion
          data.sort(sortByDate).map((cardData, index) => 
            <Card key={index} data={cardData} />
          )
```
No creo que sea buena idea usar la fecha como key para la card, puede que haya entradas que tengan el mismo dia, mejor lo veo usar el index